src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.ts

```diff
@@ -177,6 +179,8 @@ export class NavigationComponent implements OnInit, OnDestroy {
       },
       () => {},
       () => {
+        // select default daemon from the selected cluster
+        this.subs.add(this.rgwDaemonService.list().subscribe());
```

Comment:
This is general naviagtion component.
We must avoid putting specific component related stuff.
Either we pass some prop or have some other way around.

Also noticing we already have so many items here only for multicluster, which makes think its a multicluster navigation.

We should separate multicluster bits from the navigation component .
For the multicluster bits lets have separate component.

Reply:
Removed this part from this PR, will open a separate PR for this change and probably move the multi-cluster part out of the navigation component. Thanks